renderer/components/ConfigBox.tsx

```diff
@@ -85,9 +85,9 @@ export const ConfigBoxComponent = (props: Props) => {
   };
 
   const disableUploadUpdate = () => {
-    return (
-      getValidationState(ipAddress) === 'error' || getValidationState(fcAddress) === 'error' || (stationNumber < 0 && stationNumber > 4)
-    );
+    return false; // Make it false while we are using ngrok, so no IP validation is needed
+      // getValidationState(ipAddress) === 'error' || getValidationState(fcAddress) === 'error' || (stationNumber < 0 && stationNumber > 4)
+    ;
```

Comment:
ngrok will probably only be used for this year right? Maybe we can add a feature flag so that only when ngrok is enabled do we `return false`, otherwise we keep the validation.